sku.0/sys.server/compiled/game/script/library/jedi.java

```diff
@@ -2214,55 +2214,14 @@ public static void initializeCrystal(obj_id crystal, int level) throws Interrupt
     public static boolean isForceSensitive(obj_id player) throws InterruptedException
     {
         String classTemplate = getSkillTemplate(player);
-        if (classTemplate != null && classTemplate.startsWith("force"))
-        {
-            return true;
-        }
-        return false;
+        return (classTemplate != null && classTemplate.startsWith("force")) ? true : false;

```

Comment:
FYI, redundant...  you don't need logic on this at all.  Just return the stuff in the parentheses.

Reply:
Gotcha.  Thanks, I'll keep that in mind.